src/shared/components/lazyMobXTable/LazyMobXTable.spec.tsx

```diff
@@ -149,40 +149,47 @@ describe('LazyMobXTable', ()=>{
         filter: (d:any,s:string)=>(d.name.indexOf(s) > -1),
         sortBy: (d:any)=>d.name,
         render:(d:any)=>(<span>{d.name}</span>),
-        download:(d:any)=>d.name,
+        download:(d:any, s:any)=>s.name,
+        toDownload: true,
```

Comment:
I don't think we need `toDownload`, instead we can check if `download` is defined or not